packages/vats/src/issuers.js

```diff
@@ -77,7 +77,7 @@ const fromCosmosIssuerEntries = [
       bankPurse: 'Agoric staking token',
       collateralConfig: {
         keyword: 'BLD',
-        collateralValue: scaleMicro(2000000n),
+        collateralValue: scaleMicro(20000000n),
```

Comment:
in what way was this broken? I'd appreciate a more clear commit message

Reply:
Wasn't broken, just that @rowgraus wanted to change these things.  The AMM fix is in other commits that already landed on `master`.